identify/extensions.py

```diff
@@ -52,6 +52,8 @@
     'eyaml': {'text', 'yaml'},
     'feature': {'text', 'gherkin'},
     'fish': {'text', 'fish'},
+    'fit': {'binary', 'fits'},
```

Comment:
searching for this file extension produces some ambiguity so I don't think this particular one can be accepted

Reply:
So you want fits but not fit or what's the plan?

Reply:
yeah fits seems unambiguous

Reply:
Ok removed.